frontend/packages/akr/src/tests/cypress/support/page-objects/publicHomePage.ts

```diff
@@ -15,6 +19,27 @@ class PublicHomePage {
   expectText(text: string) {
     cy.should('contain.text', text);
   }
+
+  expectCookieBannerVisible() {
+    this.elements.cookieBannerContent().should('be.visible');
+  }
+
+  expectCookieBannerNotVisible() {
+    this.elements.cookieBannerContent().should('not.be.visible');
+  }
+
+  expectCookieBannerShouldNotExist() {
```

Comment:
Tämän voisi koittaa poistaa ja kutsun korvata yo. metodilla `expectCookieBannerNotVisible`

Reply:
Ja periaatteessa tänne `publicHomePage.ts` alle tehdyt muutokset voisi myös siirtää uuteen `page-objects/cookieBanner.ts` filuun. `onPublicHomePage`:lle kuitenkaan ei ole muuta käyttöä cookie bannerin cypress testeissäkään.

Reply:
Lisätty oma page-objects filu. Tuo Banneri kun operoi Dialogin kautta, niin sitä ei automaattisesti lisätä dom:iin, jolloin `not.exists` checkki pätee. Mutta jos se kerran renderöidään `open={true}` parametereillä, niin Dialogia ei poisteta domista sulkemisen jälkeen vaan se menee piiloon. Eli hyväksymisen jälkeen `is.not.visible` on sitten oikea tarkistus